lib/fog/aws/compute.rb

```diff
@@ -552,6 +557,13 @@ def _request(body, headers, idempotent, parser)
           raise case match[:code]
                 when 'NotFound', 'Unknown'
                   Fog::Compute::AWS::NotFound.slurp(error, match[:message])
+                when 'RequestLimitExceeded'
+                  if retries < max_retries
+                    sleep (2.0 ** (1.0 + retries) * 100) / 1000.0
+                    _request(body, headers, idempotent, parser, retries + 1)
```

Comment:
I would advocate for calling `retry` instead of making this a recursive function.  This would simplify error backtraces and reduce pressure on the stack.